### PR TITLE
VPC/RDS/EC2 for metrics dashboard infra

### PR DESCRIPTION
**Pull request by flamingspaz**

### CODE REVIEW

File: bitergia.tf [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    }
+}
+
+resource "aws_security_group" "bitergia-rds-sg" {
+    name        = "bitergia-rds-sg"
+    description = "Bitergia RDS SG"
+    vpc_id      = "${aws_vpc.bitergia-metrics-vpc.id}"
+}
+
+resource "aws_security_group_rule" "bitergia-rds-sg-allowmysqlfromec2" {
```
Comment (johngian) [inline bitergia.tf]: Please change this to ``bitergia-rds-sg-allowrdsfromec2``

File: bitergia.tf [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    security_group_id = "${aws_security_group.bitergia-rds-sg.id}"
+}
+
+resource "aws_security_group_rule" "bitergia-ec2-sg-allowallfromvpc" {
+    type              = "egress"
+    from_port         = 0
+    to_port           = 0
+    protocol          = "-1"
+    cidr_blocks       = ["${aws_vpc.bitergia-metrics-vpc.cidr_block}"]
+
```
Comment (johngian) [inline bitergia.tf]: Omit empty line.

File: bitergia.tf [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    vpc_id      = "${aws_vpc.bitergia-metrics-vpc.id}"
+}
+
+resource "aws_security_group_rule" "bitergia-rds-sg-allowmysqlfromec2" {
+    type              = "ingress"
+    from_port         = 3306
+    to_port           = 3306
+    protocol          = "tcp"
+    source_security_group_id = "${aws_security_group.bitergia-ec2-sg.id}"
+
```
Comment (johngian) [inline bitergia.tf]: Same here.

File: bitergia.tf [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    instance_tenancy        = "default"
+
+    tags {
+        "Name"              = "bitergia-metrics-vpc"
+        "app"               = "bitergia"
+        "env"               = "production"
+        "project"           = "metrics"
+    }
+}
+resource "aws_subnet" "bitergia-metrics-public-subnet" {
```
Comment (johngian) [inline bitergia.tf]: Please add a new line here.

File: bitergia.tf [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+  engine               = "mariadb"
+  engine_version       = "10.0.24"
+  instance_class       = "db.t2.large"
+  publicly_accessible  = false
+  backup_retention_period = 7
+  apply_immediately    = true
+  multi_az             = false
+  storage_type         = "gp2"
+  final_snapshot_identifier = "bitergia-production-db-final"
+  name                 = "bitergiaproddb"
```
Comment (johngian) [inline bitergia.tf]: Let's avoid creating a default database.

File: bitergia.tf [hunk 6]
```diff
@@ -0,0 +1,186 @@
+variable "bitergia-db_password" {}
```
Comment (johngian) [inline bitergia.tf]: Please use only ``-`` or ``_``.

Comment (flamingspaz) [inline bitergia.tf]: I do this to keep it consistent with the AWS name ("bitergia-db"). It's also done for the other DB.

File: bitergia.tf [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+    tags {
+        Name                = "bitergia"
+        app                 = "bitergia"
+        env                 = "production"
+        project             = "metrics"
+    }
+}
+
+# ElasticSearch
```
Comment (johngian) [inline bitergia.tf]: Please remove this comment.

**Comment by johngian** [comment]
r+

File: bitergia.tf [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        "Name"              = "bitergia-metrics-private-subnet"
+        "app"               = "bitergia"
+        "env"               = "production"
+        "project"           = "metrics"
+    }
+}
+
+resource "aws_internet_gateway" "bitergia-metrics-igw" {
+  vpc_id                    = "${aws_vpc.bitergia-metrics-vpc.id}"
+  tags                      = {
```
Comment (johngian) [inline bitergia.tf]: To be more uniform, lets define it as:
``
tags {
}
``

**Comment by johngian** [comment]
r+

**Pull request merged**